src/Badge/Badge.d.ts

```diff
@@ -3,9 +3,14 @@ import { HTMLAnchorAttributes } from 'svelte/elements';
 import { Color } from '../shared';
 
 export interface BadgeProps extends HTMLAnchorAttributes {
+  ariaLabel?: string;
+  border?: boolean;
```

Comment:
Minor, but this seems a slippery slope as some users may rather use shadow, etc.   The styling may be better for people to customize in their custom theme (e.g. our company theme used all UPPERCASE in badges, defined in css build).

Would this be best to define in user themes or pass via existing `class` prop?

Reply:
Users will have the ability to do both. This was meant to provide support for the most common and basic usage. I've  actually added `shadow` as an option and modified both `shadow` and `border` to accept a `boolean` or any valid class for each prop. This are optional and further customization can be done via assigning class names to the `class` prop, adjusting the theme, or not including them.